pkg/filesync/watcher.go

```diff
@@ -127,3 +128,37 @@ func start(dir string, fn func(added []string, updated []string, deleted []strin
 
 	return nil
 }
+
+// PatternMatcher is a custom implementation to match file patterns
```

Comment:
It looks like they moved the implementation into a different Go module, github.com/moby/patternmatcher. Can you use that instead?